stdlib/2/httplib.pyi

```diff
@@ -46,6 +47,8 @@ class HTTPConnection:
     timeout = ...  # type: Any
     source_address = ...  # type: Any
     sock = ...  # type: Any
+    host = ...  # type: str
+    port = ...  # type: int
```

Comment:
A style issue, mostly for future reference: We prefer PEP 526 variable annotation for new and changed code, even if the surrounding code has not been updated, yet.

Reply:
thanks. done everywhere i added stuff.